server/service/hosts.go

```diff
@@ -1682,3 +1685,58 @@ func (svc *Service) RunHostScript(ctx context.Context, request *fleet.HostScript
 
 	return nil, fleet.ErrMissingLicense
 }
+
+// //////////////////////////////////////////////////////////////////////////////
+// Get script result for a host
+// //////////////////////////////////////////////////////////////////////////////
+type getScriptResultRequest struct {
+	ExecutionID string `url:"execution_id"`
+}
+
+type getScriptResultResponse struct {
+	ScriptContents string `json:"script_contents"`
+	ExitCode       *int64 `json:"exit_code"`
+	Output         string `json:"output"`
+	Message        string `json:"message"`
+	HostName       string `json:"host_name"`
```

Comment:
Oops, I missed this before I submitted my review, but for consistency with our other API endpoints, this should be named `hostname`.

Reply:
ohhh, great catch. This will require front end changes as well, I'm thinking I could address those as part of the integration in https://github.com/fleetdm/fleet/pull/13655, what do you think?

Reply:
@roperzh yeah totally, makes sense to me!